gateway/import_handler.go

```diff
@@ -0,0 +1,29 @@
+package gateway
+
+import (
+	"net/http"
+	"strings"
+)
+
+// ImportRequestHandler is an empty struct to capture import specific request handling functionality
+type ImportRequestHandler struct{}
+
+// ProcessRequest on ImportRequestHandler splits payload by user and throws them into the webrequestQ and waits for all their responses before returning
+func (*ImportRequestHandler) ProcessRequest(gateway *Handle, w *http.ResponseWriter, r *http.Request, _ string, payload []byte, writeKey string) string {
```

Comment:
Would doing something like this make sense? Is it planned for the next iteration ?

```suggestion
type ImportRequestHandler struct{
    gateway *Handle, 
}

// ProcessRequest on ImportRequestHandler splits payload by user and throws them into the webrequestQ and waits for all their responses before returning
func (*ImportRequestHandler) ProcessRequest(w *http.ResponseWriter, r *http.Request, _ string, payload []byte, writeKey string) string {
```